compaction/compaction_manager.hh

```diff
@@ -91,7 +91,20 @@ public:
                         // counted in compaction_manager::stats::errors
         };
         static std::string_view to_string(state);
-
+        struct compaction_task_stats {
+            uint64_t fixed_validation_errors = 0;
+            uint64_t validation_errors = 0;
+
+            compaction_task_stats& operator+=(const compaction_task_stats& r) {
+                fixed_validation_errors += r.fixed_validation_errors;
+                validation_errors += r.validation_errors;
+                return *this;
+            }
+            friend compaction_task_stats operator+(compaction_task_stats l, const compaction_task_stats& r) {
```

Comment:
typically operator plus receives `const compaction_task_stats&` for the lhs side, no?
it's clearer if it copies the left hand side arg explicitly here.